account_invoice_dispatch_option/views/account_invoice_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_account_invoice_filter_inherit" model="ir.ui.view">
+        <field name="name">account.invoice.filter.inherit</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='date']" position="after">
+                <separator/>
+                <filter string="Use Company Invoice" name="use_company_invoice" domain="[('use_company_invoice', '=', True)]"/>
+                <filter string="Invoice Send Method - Post" name="invoice_send_method_post" domain="[('invoice_send_method', '=', 'post')]"/>
+                <filter string="Invoice Send Method - Email" name="invoice_send_method_email" domain="[('invoice_send_method', '=', 'email')]"/>
+                <filter string="Invoice Send Method - Do Not Send" name="invoice_send_method_do_not_send" domain="[('invoice_send_method', '=', 'do_not_send')]"/>
+                <filter string="Invoice Send Method - Others" name="invoice_send_method_others" domain="[('invoice_send_method', '=', 'others')]"/>
```

Comment:
```suggestion
　　　    <group string="Invoice Send Method">
                 <filter string="Post" name="invoice_send_method_post" domain="[('invoice_send_method', '=', 'post')]"/>
                 <filter string="Email" name="invoice_send_method_email" domain="[('invoice_send_method', '=', 'email')]"/>
                 <filter string="Do Not Send" name="invoice_send_method_do_not_send" domain="[('invoice_send_method', '=', 'do_not_send')]"/>
                 <filter string="Others" name="invoice_send_method_others" domain="[('invoice_send_method', '=', 'others')]"/>
                 </group>
```

Reply:
If I am not wrong, it will show like `Post, Email, Do Not Send, Others`. It will enough for the user to understand these are the type of invoice send method?

Reply:
I just thought grouping the filters would improve the UI.

Reply:
I am not sure I get your point but adding group with string will not be displayed like form view. The grouping is nothing for filters. And I used `separator` for these five filters.

Reply:
I was a little mistaken. Let's use the current design.